data.json

```
@@ -910,5 +910,11 @@
     "githubUsername": "amino19",
     "imageUrl": "https://user-images.githubusercontent.com/75872316/118384765-6dad0480-b626-11eb-85ce-a6af24c19181.png",
     "issueNumber": 499
-  }
+  },
+  {
+    "name": "Ayush Tamra",
+    "githubUsername": "ayushtamra",
+    "imageUrl": "https://user-images.githubusercontent.com/64156683/118395408-fb5d1400-b667-11eb-99c9-549cc5e49b00.png",
+    "issueNumber": 517
+   }
```

```suggestion
  }
```

found one little issue please align this brace with the others :)

---

Thanks for suggesting
I have resolved the required changes.